packages/contracts/src/dollar/libraries/LibUbiquityPool.sol

```diff
@@ -491,7 +491,11 @@ library LibUbiquityPool {
      */
     function collectRedemption(
         uint256 collateralIndex
-    ) internal returns (uint256 collateralAmount) {
+    )
```

Comment:
[MEDIUM] - Adding collateral enabled checks for redemption collections might make users uncapable of redeming their funds.

This PR is intended to fix [the bug described in this Sherlock issue](https://github.com/sherlock-audit/2023-12-ubiquity-judging/issues/29). However, the Sherlock report is actually wrong and the vulnerability described must not be considered as an issue in the first place, so the collateralEnabled check should not be added on collecting redemptions.

Enabling/disabling collaterals allows the protocol to add or remove supported collaterals, not to be used under uncertain situations, such as a hack. In such type of situations, the `isRedeemPaused` check can be enabled to prevent users from redeeming if something wrong is currently  happening.

The problem with adding the `collateralEnabled` check here is that if Ubiquity decides to disable a collateral in the future, some redemptions might be already queued, so users won't be able to withdraw their funds.


Recommendation: Remove the `collateralEnabled` check for collecting redemptions.